src/ProcessLauncher.php

```diff
@@ -140,7 +143,7 @@ private function startProcess(InputStream $inputStream): void
         $process->inheritEnvironmentVariables(true);
```

Comment:
env variables are always inherited and this method is deprecated since Symfony 4.4  ... 
To support Symfony 3.4 too, a method_exists might be necessary. 

```suggestion
        if(method_exists($process, 'inheritEnvironmentVariables')) {
            $process->inheritEnvironmentVariables(true);
        }
```